cmscontrib/loaders/italy_yaml.py

```
@@ -691,131 +1045,339 @@ def get_task(self, get_statement=True) -> Task | None:
+                output_template = load(conf, None, "output_template")
+                if output_template is None:
+                    output_template = "output.*"
+
+                try:
+                    input_re = compile_template_regex(input_template)
+                    output_re = compile_template_regex(output_template)
+                    paired_testcases = pair_testcases_in_directory(
+                        testcases_dir, input_re, output_re)
+                except ValueError as e:
+                    error_msg = str(e)
+                    logger.error(error_msg)
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)